.github/workflows/refapp-2x-edit-patient-demograpics.yml

```diff
@@ -0,0 +1,39 @@
+name: RefApp 2.x Edit Patient Demographics and Contact info
```

Comment:
would be better to rename the workflow to `RefApp 2.x Patient Demographics`

Reply:
@kdaud , wouldn't it be insufficient since the test is editing both demographics and contact info for the patient?

Reply:
the test implementation tells it all